plugins/Delay/DelayControlsDialog.cpp

```diff
@@ -22,10 +22,13 @@
  *
  */
 
-#include "delaycontrolsdialog.h"
-#include "delaycontrols.h"
+#include "DelayControlsDialog.h"
+#include "DelayControls.h"
 #include "embed.h"
 #include "TempoSyncKnob.h"
+#include "../Eq/EqFader.h"
```

Comment:
What is the main difference?  Can we instead toggle between the two, or are they quite different from eachother?

(Sorry for lack of insight on this)